documentation/smg/02_development_environment.md

```diff
@@ -1 +1,208 @@
+with "How do I get a user office account" three quarters of the way down the left "Contents" sidebar.
+This section has a link to a page with sign-in and registration areas.
+At the bottom right of the page, click on "Create a new facility user account with us".
+* On this registration page, fill in the form.
+Email should be your STFC email.
+For "Facilities You Use" select ISIS.
+Click "Create New Account" at the bottom.
+You will probably get an email from FacilitiesBusinessSystem@stfc.ac.uk asking you to validate your account - please do.
+* After this you should be able to log in and create a password.
+Login is your STFC email address. Your password will NOT be your federated password; you need to choose an alternative.
+You already have your federated password for access with your federated user id (form: aaa11111),
+and a UKSBS Oracle password for access with your Oracle user id (form aaaa11@ph.rc).
+Choose whatever password (existing or new) you think is appropriate for this additional login which is not the same as the other two.
+NB the federated password never seems to change but is obscure; you have to change the Oracle password every 90 days.
+
+### Opening a VM
+
+* You will see a page "Analysis Environments" hopefully open at the second tab "Analysis".
+You will see at least one choice "Excitations Single Crystal".
+Pull down `Launch` at the bottom right of the page and select "Open in Browser".
+You should get a new window with a Linux desktop.
+* In the VM desktop, the Applications menu is bottom left on a taskbar,
+followed by icons for your open windows (at the moment you have none).
+Click Applications/System to get a terminal.
+Click Software/Matlab to get a Matlab IDE window.
+Matlab will (at time of writing) be R2018b.
+* Note that the Horace item in Applications/Software is the GUI version.
+* Note that Matlab will be launched with the system Horace pre-loaded.
+To move to a development version, type `horace_off` and the `herbert_off`, change directories to the development version
+and type `herbert_init` and then `horace_init` to get Horace loaded from your git repos.
+
+## Testing your installation
+
+Check the location of your Horace by typing `horace_root`.
+
+The Matlab left window for "Current Folder" will show various directories -
+drill down into `_test/test_sqw_file`.
+At the Matlab command prompt type `s = sqw('_test/test_sqw_file/sqw_1d_1.sqw')`.
+This should run and produce some output. Type `s.data` to see the properties created.
+
+There will be an initial warning that the class `hor_config` is outdated and the default will be picked up - this is not a problem.
+A new configuration can be loaded with `hc = hor_config`
+and the values of `hc` will be printed. T
+he pixel page size in particular may be of use to modify conditions for file backing.
+
+The demo is available from the IDaaS VM by copying the directory `/mnt/babylon/Scratch/Perring/PACE_Tessella` to your home directory.
```

Comment:
Can this data be copied to the Horace SAN area so we are not referencing data on a scratch drive?

Reply:
Undoubtedly, but this is a document of what is. Guess we should raise a ticket to do the move, and then I would need to see how it can be accessed from IDaaS.

Reply:
Creating a new ticket for that sounds like the right thing to do. Can you add a reference to the new ticket in a comment on this PR